src/QsCompiler/Core/SymbolTable.fs

```diff
@@ -240,10 +240,12 @@ type private PartialNamespace private
 /// Access modifiers are taken into consideration when resolving symbols. Some methods bypass this (e.g., when returning
 /// a list of all declarations). Individual methods will mention if they adhere to symbol accessibility.
 and Namespace private
-    (name, parts : IEnumerable<KeyValuePair<NonNullable<string>,PartialNamespace>>,
-     CallablesInReferences : ImmutableDictionary<NonNullable<string>, CallableDeclarationHeader>,
-     SpecializationsInReferences : ILookup<NonNullable<string>, SpecializationDeclarationHeader * SpecializationImplementation>,
-     TypesInReferences : ImmutableDictionary<NonNullable<string>, TypeDeclarationHeader>) =
+    (name,
+     parts : IEnumerable<KeyValuePair<NonNullable<string>,PartialNamespace>>,
+     CallablesInReferences : ILookup<NonNullable<string>, CallableDeclarationHeader>,
+     SpecializationsInReferences : ILookup<NonNullable<string>,
+                                           SpecializationDeclarationHeader * SpecializationImplementation>,
```

Comment:
So this is now just a bag that contains all specializations for all callables with that (potentially internal) name?

Reply:
Yes.